src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from "react";
+
+class TodoForm extends Component {
+    constructor() {
+        super();
+        this.state = {
```

Comment:
I like that you added this state here since this form only affects this component. I agree with this. Do you feel comfortable meeting the MVP objective of having all state in App.js?